app/src/store/initialState.js

```diff
@@ -0,0 +1,59 @@
+const filterOptions = [
+    {
+        "id": 1,
+        "name": "ALL"
+    },
+    {
+        "id": 2,
+        "name": "DRAMA"
+    },
+    {
+        "id": 3,
+        "name": "ADVENTURE"
+    },
+    {
+        "id": 4,
+        "name": "HORROR"
+    },
+    {
+        "id": 5,
+        "name": "THRILLER"
+    },
+    {
+        "id": 6,
+        "name": "CRIME"
+    }
+];
+
+const sorterOptions = [

```

Comment:
why these hardcoded values should be in store? will talk today

Reply:
Fixed: placed filter and sorter options to const objects `FilterOptions` and `SorterOptions`